tailwind.config.js

```diff
@@ -4,22 +4,76 @@ import withMT from '@material-tailwind/react/utils/withMT';
 export default withMT({
   content: ['./src/**/*.{js,jsx,ts,tsx}'],
   theme: {
-    colors: {
-      primary600: '#604FED', // Base/600ID Primary bright purple
-      primary700: '#5648CE',
-      primary25: '#F9F9FF', // Light purple
-      lightBlue: '#363F72',
-      lightBlue800: '#424974',
-      lightBlue700: '#5A6192',
-      lightBlue100: '#DDDFF3',
-      lightBlue200: '#D5D9EB',
-      darkPurple: '#362644',
-      base900DarkPurple: '#302935', // Text, headings
-      ashGrey400: '#8B94A6', // Disabled buttons
-      error: '#FEE4E2',
-      white: '#FFFFFF',
+    extend: {
+      colors: {
+        // Light purple
+        primary: {
+          25: '#F9F9FF',
+          50: '#F1EFFF',
+          100: '#E9E7FF',
+          200: '#DAD7FD',
+          300: '#AFA6FB',
+          400: '#9083FF',
+          500: '#7969FF',
+          600: '#604FED', // Base/600ID Primary bright purple
+          700: '#4F3ED8',
+          800: '#382AAD',
+          900: '#271D7B',
+        },
+        lightBlue: {
```

Comment:
I realize we also used camelCase before, but other projects use `light-blue` etc